.gitignore

```diff
@@ -0,0 +1,2 @@
+notebooks/.ipynb_checkpoints/*
+__pycache__
```

Comment:
added generic files to gitignore 😄

Reply:
ok, I suppose you can ignore the entier `.ipynb_checkpoints` directory